script.js

```diff
@@ -52,11 +51,24 @@ restaurantsApp.getReviews=function(result){
         })
     }}
 
-restaurantsApp.displayRestaurantDetails = function(result) {
-  
-    $('.ListOne').empty()
+// Use the GeoLocation-DB API to get the user's city
+restaurantsApp.getCity = async function() {
+    // wait for the result to come back (promise)
+    let location = await $.ajax({
+        url: "https://geolocation-db.com/jsonp",
+        jsonpCallback: "callback",
+        dataType: "jsonp"
+    });
+
+    return location.city;
+}
+
+restaurantsApp.displayRestaurantDetails = function(result, n) {
```

Comment:
can you give `n` a more descriptive name? it's hard to know what it is supposed to represent right now.